client/src/views/UserView.tsx

```diff
@@ -1,44 +1,91 @@
 import React from 'react'
-import {
-  TableContainer,
-  Table,
-  TableHead,
-  TableBody,
-  TableCell,
-  TableRow,
-  Typography,
-  Paper,
-  Container,
-} from '@material-ui/core'
+import { Container } from '@material-ui/core'
 
-import { UserRow } from '../components/UserRow'
-import { useUsers } from '../hooks/useUsers'
+import { useUsers, User } from '../hooks/useUsers'
+import MaterialTable from 'material-table'
+import UserAvatar from '../components/UserAvatar'
+import { getClanForId } from '../utils/clanUtils'
+import { UserChip } from '../components/UserChip'
+import { useHistory } from 'react-router-dom'
+
+interface RowUser {
+  user: User
+  discordName: string
+  jigokuName: string
+  preferredClan: string
+  userId: string
+}
 
 export function UserView(): JSX.Element {
-  const users = useUsers()
+  const users: RowUser[] = useUsers().flatMap((user: User) => {
```

Comment:
`Array#flatMap` is still experimental, and not supported by Microsoft Edge. For the current use case `Array#map` should be enough, and it has better support.

Reply:
Also, maybe this transformation should be inside `useUsers`? The hook returning the objects already mapped and ready to use?